.github/workflows/ci.yml

```diff
@@ -77,12 +77,7 @@ jobs:
       - name: Clippy
         run: cargo clippy --all-targets --locked -- -D warnings
       - name: SQLFluff
-        run: pipx run sqlfluff==3.3.0 lint
-      - name: Verifying expression dependencies workspace
-        run: |
-          rustup toolchain install nightly
-          chmod +x ./.scripts/check-expression-deps.rs
-          ./.scripts/check-expression-deps.rs
```

Comment:
delete the script then (or share code between the new test and the script)

Reply:
[ce97279](https://github.com/geo-engine/geoengine/pull/1048/commits/ce972795010c5b783bb6e508061ce0a588c4d93d)